src/test/scala/org/ergoplatform/ErgoSanity.scala

```diff
@@ -15,23 +15,22 @@ import org.scalacheck.Gen
 import scorex.core.bytesToId
 import scorex.core.transaction.state.MinimalState
 import scorex.crypto.authds.ADDigest
-import scorex.crypto.hash.Digest32
+import scorex.crypto.hash.{Blake2b256, Digest32}
+import scorex.testkit.generators.{ModifierProducerTemplateItem, SynInvalid, Valid}
 import scorex.testkit.properties._
-import scorex.testkit.properties.mempool.MempoolTransactionsTest
+import scorex.testkit.properties.mempool.{MempoolFilterPerformanceTest, MempoolRemovalTest, MempoolTransactionsTest}
 import scorex.testkit.properties.state.StateApplicationTest
 import scorex.utils.Random
 
-//todo: currently this class parametrized with UtxoState, consider DigestState as well
 trait ErgoSanity[ST <: MinimalState[PM, ST]] extends HistoryTests[TX, PM, SI, HT]
   with StateApplicationTest[PM, ST]
-  //with StateApplyChangesTest[P, TX, PM, B, ST]
   //with WalletSecretsTest[P, TX, PM]
-  //with StateRollbackTest[P, TX, PM, B, ST, SI, HT, MPool]
   with MempoolTransactionsTest[TX, MPool]
   // todo: convert MempoolFilterPerformanceTest to benchmark
```

Comment:
Lets fix this todo - move performance tests to a separate benchmark that won't be started by default on push.